src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -64,6 +64,13 @@ public static void ImaginaryOne()
             VerifyMagnitudePhaseProperties(Complex.ImaginaryOne, 1, Math.PI / 2);
         }
 
+        [Fact]
+        public static void SqrtMinusOne()
+        {
+            // Sqrt(-1) should be I exactly, not just tollerably close.
```

Comment:
~~tollerably~~ tolerably. Actually, I would just remove this comment, the next line speaks for itself, and we don't necessarily need any references to the old behavior that we are removing.